inspectors/view/qml/notation/NotationInspectorView.qml

```diff
@@ -223,5 +224,43 @@ FocusableItem {
                 staffSettingsModel: root.model ? root.model.staffSettingsModel : null
             }
         }
+
+        FlatButton {
+            id: sectionBreakSettingsButton
+
+            icon: IconNameTypes.SECTION_BREAK
```

Comment:
I recommend that you merge my PR with the UI module and use the types from it
https://github.com/igorkorsukov/MuseScore/pull/7